src/components/TimeCardPage/TimeSheet.tsx

```diff
@@ -61,21 +122,36 @@ export default function Page() {
         setCurrentTimesheetsToDisplay (userTimesheets, date); 
     }
     
-    const [userTimesheets, setUserTimesheets] = useState([]); 
-    const [currentTimesheets, setCurrentTimesheets] = useState([]);
-    const [selectedTimesheet, setTimesheet] = useState();
+    //TODO: default to first employee but idk if employee always exists
+    const [selected, setSelected] = useState<UserSchema>();
+    const [user, setUser] = useState<UserSchema>();
+
+    const [userTimesheets, setUserTimesheets] = useState<TimeSheetSchema[]>([]); 
+    const [currentTimesheets, setCurrentTimesheets] = useState<TimeSheetSchema[]>([]);
+    const [selectedTimesheet, setTimesheet] = useState<TimeSheetSchema>();
     const columns = defaultColumns 
 
-    //Pulls user timesheets, marking first returned as the active one
+    // Pulls user timesheets, marking first returned as the active one
     useEffect(() => {
         // Uncomment this if you want the default one loaded 
         //setTimesheet(testingTimesheetResp)
+        apiClient.getUser().then(userInfo => {
+            setUser(userInfo);
+        })
         apiClient.getUserTimesheets().then(timesheets => {
             setUserTimesheets(timesheets); 
             //By Default just render / select the first timesheet for now 
-            setCurrentTimesheetsToDisplay (timesheets, startDate); 
+            setCurrentTimesheetsToDisplay(timesheets, startDate);
+            // fetch the information of the user whos timesheet is being displayed
+            // if user is an employee selected and user would be the same
+            // if user is a supervisor/admin then selected would contain the information of the user
```

Comment:
Good explanation - can we move this up or restate some of it where the state variables are declared?